vestings/addresses.py

```diff
@@ -0,0 +1,24 @@
+from constants import *
+
+def get_airdrop_addresses(chain_id):
+    return {
+        "user": {
+            1: MAINNET_USER_AIRDROP_ADDRESS,
+            4: RINKEBY_USER_AIRDROP_ADDRESS,
+            5: GOERLI_USER_AIRDROP_ADDRESS
+        }[chain_id],
+        "user_v2": {
+            1: MAINNET_USER_AIRDROP_ADDRESS,
+            4: RINKEBY_USER_AIRDROP_ADDRESS,
```

Comment:
do we need Rinkeby ? WE are not going to deploy test data there , as there is no support for Rinkeby anymore